### DISCUSSION TITLE
Spring Boot 3 support

### DISCUSSION THREAD

**Discussion by davidmestr** [infinispan/infinispan#10692] [General]
When it is planned to have a version with support for Spring Boot 3?

**Comment by tristantarrant** [comment]
https://issues.redhat.com/browse/ISPN-14270
https://issues.redhat.com/browse/ISPN-13424

14.0.7.Final should be out tomorrow

**Reply by srcimon** [reply]
Hello @tristantarrant , I tried 14.0.7.FINAL but failed. I created a small repository to recreate the issue: https://github.com/simonbas/inifinispan-spring-boot-3-issue

Did I miss something from https://infinispan.org/docs/14.0.x/titles/spring_boot/starter.html?

**Reply by tristantarrant** [reply]
You need to use `infinispan-spring-boot3-starter`

**Reply by srcimon** [reply]
Thank you very much
</img href="./head-wall-bang.jpg">

**Reply by tristantarrant** [reply]
Not your fault: we still need to update the docs
